Footprints/SparkFun-Semiconductor-Standard.pretty/PowerVFDFN-13.kicad_mod

```diff
@@ -1,4 +1,4 @@
-(footprint "V-DFN3020-13"
```

Comment:
Also need to update the symbol to use the new footprint name

Reply:
Yep. Still working my way through, but catch it if I don't.